.github/workflows/postman-build-and-publish.yml

```diff
@@ -71,7 +71,11 @@ jobs:
           push: true
           tags: |
             ${{ env.IMAGE_NAME }}:${{ env.COMMIT_TAG }}-${{ env.UNTESTED_TAG_SUFFIX }}
-          cache-from: type=registry,ref=${{ env.IMAGE_NAME }}:buildcache
-          cache-to: type=registry,ref=${{ env.IMAGE_NAME }}:buildcache,mode=max
+          cache-from: |
+            type=registry,ref=${{ env.IMAGE_NAME }}:buildcache-amd64,platform=linux/amd64
```

Comment:
What was the benefit of specifying the platform for the cache? reduced the cache size? Also as of now (before the changes here), the cache is working for both platforms, right?

Reply:
The cache was not working properly for multi platforms. We were using the same cache for both platform. Apparently the cache is computed differently on each platform. So I added a cache for each platform